JavaScript/JavaScript.sublime-syntax

```diff
@@ -1834,9 +1834,9 @@ contexts:
       captures:
         1: punctuation.definition.variable.js
         2: variable.other.readwrite.js
-
+      pop: true
```

Comment:
How about moving the whole pattern upwards to achieve the same order as in `method-name` context?

_Both contexts seem to match the same tokens but only apply different scopes. It would therefore probably make sense to align their formal content._

Reply:
Done.